client/src/components/infopanel/utdypendeelementer/Friskmelding.tsx

```diff
@@ -5,116 +5,70 @@ import { tilLesbarDatoMedArstall } from '../../../utils/datoUtils';
 
 import SeksjonMedTittel from '../layout/SeksjonMedTittel';
 
-import tekster from '../infopanel-tekster';
-import Margin from '../layout/Margin';
 import ElementMedTekst from '../layout/ElementMedTekst';
-import EnkelCheckbox from '../layout/EnkelCheckbox';
 
 interface FriskmeldingProps {
-    prognose?: Prognose;
+  prognose?: Prognose;
 }
 
 const Friskmelding = ({ prognose }: FriskmeldingProps) => {
-    // TODO: legg til logikk for visning av seksjon
-    /*
-    const visSeksjon = (sykmelding.friskmelding.antarReturSammeArbeidsgiver ||
-    sykmelding.friskmelding.antattDatoReturSammeArbeidsgiver ||
-    sykmelding.friskmelding.antarReturAnnenArbeidsgiver ||
-    sykmelding.friskmelding.tilbakemeldingReturArbeid ||
-    sykmelding.friskmelding.utenArbeidsgiverTilbakemelding ||
-    sykmelding.friskmelding.utenArbeidsgiverAntarTilbakeIArbeid ||
-    sykmelding.friskmelding.utenArbeidsgiverAntarTilbakeIArbeidDato ||
-    sykmelding.friskmelding.utenArbeidsgiverTilbakemelding);
-    */
-    if (!prognose) {
-        return null;
-    }
+  if (!prognose) {
+    return null;
+  }
 
-    const { erIArbeid, erIkkeIArbeid, hensynArbeidsplassen, arbeidsforEtterPeriode } = prognose;
```

Comment:
disse to er en del av prognose-objektet, så burde nok være med for "hele sykmeldingen" selv om det ikke er i skissen